### DISCUSSION TITLE
[Fixed] Website need an active maintainer

### DISCUSSION THREAD

**Discussion by paulcarroty** [VSCodium/vscodium#965] [Q&A] [answered]
A funny month+ old issue with missing space (#925) and 6 [pull requests](https://github.com/VSCodium/vscodium.github.io/pulls) waiting since September and longer - this is sad, let's add some movement. Cast @stripedpajamas @daiyam @GitMensch.

**Comment by stripedpajamas** [comment, marked as answer]
You're right, I'm not sure how I missed the notifications from that repo. May have inadvertently muted them. I'm looking through the PRs now.

**Reply by paulcarroty** [reply]
Thanks.

**Comment by daiyam** [comment]
Ya, I've seen those but sadly, I'm not a maintainer for the website.

**Comment by GitMensch** [comment]
Thank you for the ping! @stripedpajamas Thanks for the trust to add me to that repo, too (couldn't do anything before myself either).
Current state is that all but one merge-request are pulled, I'd like one "ok" for the last one (conflicts solved).

It will be interesting to see how the work on that repo and our main site will go on.

**Comment by daiyam** [comment]
@stripedpajamas Thanks Peter.
